components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -167,6 +170,13 @@ public void handle(HttpServletRequest request, HttpServletResponse response) thr
                             log.debug("No authentication request found in the cache for sessionDataKey: "
                                     + sessionDataKey);
                         }
+                        if (LoggerUtils.isDiagnosticLogsEnabled()) {
+                            LoggerUtils.triggerDiagnosticLogEvent(
+                                    FrameworkConstants.LogConstants.AUTHENTICATION_FRAMEWORK, null,
+                                    LogConstants.SUCCESS, "No authentication request found in the " +
+                                            "cache for sessionDataKey: " + sessionDataKey, "handle-authentication" +
```

Comment:
feels this is more technical, cache is an internal thing. so, better to reword this to convey what happened from external user pov

Reply:
This has been removed with https://github.com/wso2/carbon-identity-framework/pull/4326/commits/ca34cc4be2ac20f9c870f1ded78fab17d9bf6d94 since this information is internal and not relevant for the user.